main_server/main_server.js

```diff
@@ -28,10 +29,35 @@ var session = require('express-session')({
 
 app.use(session);
 var socketSession = require('express-socket.io-session')
-
-var config_details = require('/etc/main_server/conf.json');
 var mysql = require('mysql');
 
+const { check } = require('../util/environmentCheck.js');
+/* The environment variables MSCONFIG, MSLABCONFIG, MSSCORES and MSAPIKEYS
+  will contain the path to the respective config files.
+  The paths for the config files are
+  1.Variable: MSCONFIG
+    Actual Path: "../deploy/configs/main_server/conf.json"
+    Path in docker containers: "/etc/main_server/conf.json"
+
+  2.Variable: MSLABCONFIG
+    Actual Path: "../deploy/configs/main_server/labs.json"
+    Path in docker containers: "/etc/main_server/labs.json"
+
+  3.Variable: MSCOURSECONFIG
+    Actual Path: "../deploy/configs/main_server/course.json"
+    Path in docker containers: "/etc/main_server/course.json"
+
+  4.Variable: MSAPIKEYS
+    Actual Path: "../deploy/configs/main_server/APIKeys.json"
+    Path in docker containers: "/etc/main_server/APIKeys.json"
+*/
+
+check('MSCONFIG');
```

Comment:
The check function is checking the definition of the environment variable. It can also check for the existence of the file and the read permissions. Once the existence and permissions checks pass, we can subject the configuration file to validity check by iterating the isNodeValid() function over all the entries in the configuration file.

This is just an idea. Please suggest alternatives and improvements.

Reply:
@AnkshitJain 
The scope of the above suggestion is very big. I have opened a new issue #250 to keep track of the suggestion. Let's wrap up this PR with the existing check() functionality.